dallinger/recruiters.py

```diff
@@ -381,6 +487,68 @@ def _handle_exit_form_submission(self, assignment_id: str, participant_id: str):
         q = _get_queue()
         q.enqueue(worker_function, "AssignmentSubmitted", assignment_id, participant_id)
 
+    def load_service(self, sandbox):
+        return _prolific_service_from_config(sandbox)
+
+    def clean_qualifications(self, experiment_details):
```

Comment:
This is a classic example of a function that should have an automated test

Reply:
You could have some examples of real Prolific qualifications to clean

Reply:
Yep, added a test with some anonymized test data.

Reply:
Great!